### DISCUSSION TITLE
font size in menu adjustable

### DISCUSSION THREAD

**Discussion by On-3** [brackets-cont/brackets#298] [Feature Request]
Just started using this editor and noticed right away that while I can adjust the font of the code that I type the font of the menus on the side and the top remain fixed. I find them a bit small. Would be great if those can be adjusted as well

**Comment by charlypa** [comment]
@On-3 

We've updated our editor to Phoenix Code, launched just yesterday. It now supports native M1 and Intel dmg files, moving beyond the older Brackets x86 binaries. You can download it at https://phcode.io/.


use `ctrl +` and ` ctrl -` to pdate entire UI of Phoenix Code 

use `ctrl shift +` and `ctrl shift -` to Update just fonts of Phoenix Code
